README.md

```diff
@@ -593,15 +593,17 @@ that are common to all storage systems are omitted for brevity.
 | **Features**                | Gaia | [Sia](https://sia.tech/) | [Storj](https://storj.io/) | [IPFS](https://ipfs.io) | [DAT](https://datproject.org) | [SSB](https://www.scuttlebutt.nz/) |
 |-----------------------------|------|--------------------------|----------------------------|-------------------------|-------------------------------|------------------------------------|
 | User controls where data is hosted           | X |   |   |   |   |   |
-| Data can be viewed in a normal Web browser   | X |   |   | X |   |   |
+| Data stored redundantly                      | optional | X | X | optional | optional | optional |
 | Data is read/write                           | X |   |   |   | X | X |
-| Data can be deleted                          | X |   |   |   | X | X |
+| Data can be deleted                          | X | X |   |   | X | X |
 | Data can be listed                           | X | X | X |   | X | X |
 | Deleted data space is reclaimed              | X | X | X | X |   |   | 
-| Data lookups have predictable performance    | X |   | X |   |   |   |
+| Data lookups have predictable performance    | X | X | X |   |   |   |
```

Comment:
I don't believe this is true in general, your (single data point of) experiences not withstanding.  Does Sia offer SLAs?  Is it even possible in the system design of Sia to _offer_ SLAs?

Reply:
>Using the Sia blockchain, renters form file contracts with hosts. These contracts set pricing, uptime commitments, and other aspects of the relationship between the renters and the hosts.
File contracts are a type of smart contract. They allow us to create cryptographic service level agreements (SLAs) that are stored on the Sia blockchain.
Since file contracts are automatically enforced by the network, Sia has no need for intermediaries or trusted third parties.

Source: https://sia.tech/technology

If the SLA fails, the host will lose the locked collateral. And of course renters prefer hosts with moer collateral.

Reply:
Thanks for the link.

The reason I asked the second question is because while it's all well and good for Sia to _claim_ to offer SLAs, I don't think the design of the system itself makes this possible to do beyond a binary outcome.  The SLA Sia seems to offer, from what I could gather from their website, is that either

* the file is available if you wait long enough, or
* the file is not available; slash the miner

This is not what I would call "predictable performance," as understood by a reasonable user.  "Predictable performance" here means "I can request the file from any computer in the world and at any time, and it will almost always download in the same reasonable amount of time."  For example, Dropbox has predictable performance -- you can request to download a 1GB file from anywhere in the world at any time from any device, and there's a good chance (e.g. 99% or higher) it will (1) be available and (2) download relatively quickly -- as fast as your Internet connection allows.  As a counterexample, BitTorrent does not have predictable performance -- the download times and availability of a file is highly variable, and depends on the number of seeders that are online.  Unless the file is well-seeded, you won't saturate your network bandwidth either.

Under this understanding, does Sia provide predictable performance similar to e.g. Dropbox?  Or is it more like BitTorrent?  From what I have been able to find online, it looks more like BitTorrent.

I'd be willing to alter Gaia's `X` here to say something like `depends on the storage back-end`, because the read performance of Gaia is wholly dependent on the storage provider (which itself can offer less-than-predictable performance, if the user so chooses).

Reply:
No blockchain protocols can offer SLA's. Simple fact is, SLA's are more than just performance and availability guarantees; Support, penalties, customization, security, compliance and accountability are all part of SLA's.

Regarding performance, you guys are looking at this the wrong way. Sia is a storage marketplace with a vast number of different storage nodes with different performance metrics. It is not a storage service provider, that is layer two, e.g. a service built on Sia. 

If you are layer two and you have your own storage node selection layer built on Sia with benchmarking etc. you **will** get predictable performance. If you use the siad or the UI offered by Sia, not so much.

So if we talk strictly the API's offered by Sia @jcnelson is correct.

Reply:
> Regarding performance, you guys are looking at this the wrong way. Sia is a storage marketplace with a vast number of different storage nodes with different performance metrics. It is not a storage service provider, that is layer two, e.g. a service built on Sia.

To avoid future confusion, is there a different name for Sia-the-widely-used-blockchain-network than simply "Sia?"  Up until now, I thought the word "Sia" could be used to refer to either the protocol and the widely-used blockchain network.  This is the case for other p2p protocols, for example -- BitTorrent is the protocol, but The Pirate Bay is a particular BitTorrent network (with a particular collection of torrents and peers).

But to be clear, I think this table is correct for Sia regardless of whether you interpret it to mean the protocol or the blockchain network.

> If you are layer two and you have your own storage node selection layer built on Sia with benchmarking etc. you will get predictable performance. If you use the siad or the UI offered by Sia, not so much.

Are there such widely-used systems?  Happy to add columns for them, based on their particular feature sets.

Reply:
I disagree. Comparing services as Gaia that use centralized storage providers to blockchain protocols like Sia does not make any sense. There are a few services that have built AWS S3 API's on Sia like https://filebase.com/ which do offer predictable performance.

Reply:
What guarantees does Filebase make that Sia does not?  Can I measure them or audit them?  If Filebase is a distinct storage medium from Sia that makes different guarantees about e.g. redundancy, then it would deserve its own column.